kms/keys.js

```diff
@@ -0,0 +1,1467 @@
+              console.log(`${binding.role}:`);
+              binding.members.forEach((member) => {
+                console.log(`  ${member}`);
+              });
+            }
+          });
+        } else {
+          console.log(`Policy for crypto key ${keyName} is empty.`);
+        }
+      });
+    });
+  });
+  // [END kms_remove_member_from_cryptokey_policy]
+}
+
+function createCryptoKeyVersion (projectId, location, keyRingName, keyName) {
+  // [START kms_create_cryptokey_version]
+  // Your Google Cloud Platform project ID
+  // const projectId = 'YOUR_PROJECT_ID';
+
+  // The location of the crypto key versions's key ring, e.g. "global"
+  // const location = 'global';
+
+  // The name of the crypto key version's key ring, e.g. "my-key-ring"
+  // const keyRingName = 'my-key-ring';
+
+  // The name of the version's crypto key, e.g. "my-key"
+  // const keyName = 'my-key';
+
+  // Builds and authorizes a Cloud KMS client
+  buildAndAuthorizeService((err, cloudkms) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+
+    const request = {
+      // This will be a path parameter in the request URL
+      parent: `projects/${projectId}/locations/${location}/keyRings/${keyRingName}/cryptoKeys/${keyName}`
+    };
+
+    // Creates a new crypto key version
+    cloudkms.projects.locations.keyRings.cryptoKeys.cryptoKeyVersions.create(request, (err, cryptoKeyVersion) => {
+      if (err) {
+        console.log(err);
+        return;
+      }
+
+      console.log(`Crypto key version ${cryptoKeyVersion.name} created.`);
+    });
+  });
+  // [END kms_create_cryptokey_version]
+}
+
+function listCryptoKeyVersions (projectId, location, keyRingName, keyName) {
+  // [START kms_list_cryptokey_versions]
+  // Your Google Cloud Platform project ID
+  // const projectId = 'YOUR_PROJECT_ID';
+
+  // The location of the crypto key's key ring, e.g. "global"
+  // const location = 'global';
+
+  // The name of the crypto key's key ring, e.g. "my-key-ring"
+  // const keyRingName = 'my-key-ring';
+
+  // The name of the crypto key from which to list versions, e.g. "my-key"
+  // const keyName = 'my-key-ring';
+
+  // Builds and authorizes a Cloud KMS client
+  buildAndAuthorizeService((err, cloudkms) => {
+    if (err) {
+      console.log(err);
+      return;
+    }
+
+    const request = {
+      // This will be a path parameter in the request URL
+      parent: `projects/${projectId}/locations/${location}/keyRings/${keyRingName}/cryptoKeys/${keyName}`
+    };
+
+    // Creates a new key ring
+    cloudkms.projects.locations.keyRings.cryptoKeys.cryptoKeyVersions.list(request, (err, result) => {
+      if (err) {
+        console.log(err);
+        return;
+      }
+
+      const cryptoKeyVersions = result.cryptoKeyVersions || [];
+
+      if (cryptoKeyVersions.length) {
+        cryptoKeyVersions.forEach((version) => {
+          console.log(`${version.name}:`);
+          console.log(`  Created: ${new Date(version.createTime)}`);
```

Comment:
Maybe, but I didn't like how huge the tab characters were when printed to my console. 2 spaces was more reliable.